### ISSUE TITLE
"Del" key doesn't delete files while using the Spanish translation.

### ISSUE THREAD

**Issue by d3p4z** [lxqt/pcmanfm-qt#1622]
##### Expected Behavior
Pressing "Del" key should delete files.

##### Current Behavior
"Del" key doesn't delete files while using the Spanish translation.

##### Possible Solution
Removing the following message from "pcmanfm/translations/pcmanfm-qt_es.ts" solves the issue here:
```
<message>
    <location filename="../main-win.ui" line="654"/>
    <source>Del</source>
    <translation>Suprimir</translation>
</message>
```
Though I don't know if that message is supposed to translate anything relevant.

##### Steps to Reproduce (for bugs)
1. Set "LANG" to "es"
2. Launch pcmanfm-qt and try to delete a file using the "Del" key

##### System Information
* PcManFm-qt version: 1.1.0
* Distribution & Version: Void Linux
* Qt Version: 5.15.4+20220606_1
* liblxqt Version: 1.1.0_1

**Comment by tsujan** [comment]
I think the translation is fixed. @stefonarch, is it?

**Comment by stefonarch** [comment]
I remember that there _was_ a wrong translation, but I see 
1. https://translate.lxqt-project.org/translate/lxqt-desktop/pcmanfm-qt/es/?offset=&q=+source%3A%3DDel+&sort_by=-priority%2Cposition&checksum=
2. `Suprimir` works here:
![screenshot-fd6acad4](https://user-images.githubusercontent.com/10681413/181523978-ef4d7b1f-6421-4cb9-88fb-eda91a9ace7e.png)

**Comment by tsujan** [comment]
> Suprimir works here.

Strange! It doesn't appear and doesn't work here, with `LANGUAGE=es pcmanfm-qt`:

![es](https://user-images.githubusercontent.com/981076/181525436-61a74fb9-cf9c-48f5-a80c-b499f2d80ef3.png)

**Comment by stefonarch** [comment]
Found the issue: https://github.com/lxqt/pcmanfm-qt/issues/394#issuecomment-1159484872 it was "Eliminar" first.

So the question is: why? @d3p4z Do you see the translation in the menu es above?

**Comment by tsujan** [comment]
Qt version is a factor too. Mine is 5.15.5+kde+r173, from Arch.

**Comment by stefonarch** [comment]
I fired up Lubuntu  22.04 LXQt 1.1.0 with the new ppa and no:
![screenshot-f5497070](https://user-images.githubusercontent.com/10681413/181531449-048d64ea-ff7a-4c09-9e44-f3380e58ec9d.png)

Qt here is 5.15.5+kde+r172-1 
 Lubuntu Qt version 5.15.3

**Comment by d3p4z** [comment]
This is how I see it before removing the message I mentioned in my original post:
![before](https://user-images.githubusercontent.com/32247963/181534113-ba326a58-79d5-43c0-97b3-f1349c30b5e8.png)

And here's how I see it after removing such message:
![after](https://user-images.githubusercontent.com/32247963/181534914-0a932299-918a-4adf-9c14-44fd8c550510.png)

I took a look to the German translation, and that message isn't translated at all?

**Comment by tsujan** [comment]
> I fired up Lubuntu 22.04 LXQt 1.1.0 with the new ppa and no:

Oh, sorry, my translation is old. It's `Supr` here, not `Suprimir`. It should be the same in LXQt 1.1.0.

**Comment by stefonarch** [comment]
In German it's `Entf` (=Entfernen). and working.
But to be sure: you rebuilt the .qm file after deleting the line?

**Comment by d3p4z** [comment]
Yeah, in German it's working perfectly fine and their translation doesn't need that <message>.
I deleted the line in the sources and recompiled pcmanfm-qt from the start.
Seems that Qt itself translates it to Suprimir.

**Comment by tsujan** [comment]
@d3p4z 
If your provided info is correct, you have pcmanfm-qt 1.1.0, which doesn't have `Suprimir` but `Supr`.

**Comment by tsujan** [comment]
@stefonarch 
The translation of each shortcut can be disabled in the ui files. But I'm not sure if it's a good idea with *all* languages (is it?). IMHO, translators should treat shortcuts with more care.

**Comment by d3p4z** [comment]
@tsujan 
Yeah, I have pcmanfm-qt 1.1.0, does have "Supr" instead of "Suprimir". 
It's certainly fixed in git. I patched my 1.1.0 copy to use "Suprimir" for the time being until there's a new release and Void packages it.

Thank you, and sorry if I wasted anyone's time.

**Comment by stefonarch** [comment]
> Thank you, and sorry if I wasted anyone's time.

It's fine, I remembered it was a longer time ago fixed and present in 1.1
